src/platform/plugins/shared/discover/public/context_awareness/profile_providers/security/components/alert_event_overview.tsx

```diff
@@ -0,0 +1,133 @@
+import type { FC, PropsWithChildren } from 'react';
+import { getFieldValue } from '@kbn/discover-utils';
+import type { DocViewerComponent } from '@kbn/unified-doc-viewer/src/services/types';
+import {
+  EuiTitle,
+  EuiSpacer,
+  EuiAccordion,
+  EuiButton,
+  EuiFlexGroup,
+  EuiFlexItem,
+  EuiText,
+} from '@elastic/eui';
+import * as i18n from '../translations';
+import { getEcsAllowedValueDescription, getSecurityTimelineRedirectUrl } from '../utils';
+import { useDiscoverServices } from '../../../../hooks/use_discover_services';
+
+export const ExpandableSection: FC<PropsWithChildren<{ title: string }>> = ({
+  title,
+  children,
+}) => {
+  const [trigger, setTrigger] = useState<'open' | 'closed'>('open');
+
+  const onToggle = (isOpen: boolean) => {
+    const newState = isOpen ? 'open' : 'closed';
+    setTrigger(newState);
+  };
+
+  return (
+    <EuiAccordion
+      id={`accordion-${title}`}
+      forceState={trigger}
+      onToggle={onToggle}
+      buttonContent={
+        <EuiTitle size="xs" data-test-subj={`expandableHeader-${title}`}>
+          <h4>{title}</h4>
+        </EuiTitle>
+      }
+    >
+      <EuiSpacer size="m" />
+      <EuiFlexGroup
+        gutterSize={'m'}
+        direction="column"
+        data-test-subj={`expandableContent-${title}`}
+      >
+        {children}
+      </EuiFlexGroup>
+    </EuiAccordion>
+  );
+};
+
+export const AlertEventOverview: DocViewerComponent = ({ hit }) => {
+  const {
+    application: { getUrlForApp },
+  } = useDiscoverServices();
+
+  const timelinesURL = getUrlForApp('securitySolutionUI', {
+    path: 'alerts',
+  });
+
+  const reason = useMemo(() => getFieldValue(hit, 'kibana.alert.reason') as string, [hit]);
+  const description = useMemo(
+    () => getFieldValue(hit, 'kibana.alert.rule.description') as string,
+    [hit]
+  );
+  const alertURL = useMemo(() => getFieldValue(hit, 'kibana.alert.url') as string, [hit]);
+  const eventKind = useMemo(() => getFieldValue(hit, 'event.kind') as string, [hit]);
+  const isAlert = useMemo(() => eventKind === 'signal', [eventKind]);
+  const eventId = useMemo(() => getFieldValue(hit, '_id') as string, [hit]);
+  const eventURL = useMemo(
+    () =>
+      getSecurityTimelineRedirectUrl({
+        from: getFieldValue(hit, '@timestamp') as string,
+        to: getFieldValue(hit, '@timestamp') as string,
+        eventId: eventId as string,
+        index: getFieldValue(hit, '_index') as string,
+        baseURL: timelinesURL,
+      }),
+    [hit, eventId, timelinesURL]
+  );
+
+  const eventCategory = useMemo(() => getFieldValue(hit, 'event.category') as string, [hit]);
+
+  return (
+    <EuiFlexGroup
+      data-test-subj={isAlert ? 'alertOverview' : 'eventOverview'}
+      gutterSize="m"
+      direction="column"
+      style={{ paddingBlock: '20px' }}
+    >
+      <EuiFlexItem>
+        <ExpandableSection title={'About'}>
+          <EuiText data-test-subj="about">{getEcsAllowedValueDescription(eventCategory)}</EuiText>
```

Comment:
I don't know if you had mocks, but to me, the size of the text here and the 2 sections below looks really big compared to the rest of the Discover app...

![Screenshot 2025-06-23 at 11 19 54 AM](https://github.com/user-attachments/assets/47494ba0-aec4-4559-91c1-9e1c0f751967)

I find that it looks better with `size="s"` but this is very subjective...
![Screenshot 2025-06-23 at 11 19 23 AM](https://github.com/user-attachments/assets/267a211f-40bd-4ac8-bb14-95ab92d8da7f)